src/internal.c

```diff
@@ -20535,6 +20535,61 @@ const char* GetCipherNameInternal(const byte cipherSuite0, const byte cipherSuit
 }
 
 #if defined(WOLFSSL_QT) || defined(OPENSSL_ALL)
+/* Segment cipher name into n[n0,n1,n2,n4]
+ * @param cipher a pointer to WOLFSSL_CIPHER
+ * @param n return segment cipher name
+ * return cipher name if cipher is in the list,
+ *        otherwise NULL
+ */
+const char* GetCipherSegment(const WOLFSSL_CIPHER* cipher, char n[][MAX_SEGMENT_SZ])
+{
+    int i,j,k;
+    int strLen;
+    unsigned long offset;
+    const char* name;
+    
+    /* sanity check */
```

Comment:
It seems like we should also check that "n" is not NULL

Reply:
That is correct. Added check that "n" is not NULL